### PR TITLE
changes to fix radio buttons not working properly

### LINKED ISSUE #119: Parameter button does cause editor to ask for value
After filling out all the fields of a new quantity node, I click th eparameter button, and nothing happens.  It shoudl open fields for value and units.  when I close the node and reopen it, the menu has reset to be unknown.

### PR DESCRIPTION
**Pull request by sachingrover211**
issue exists only on windows+chrome on certain machines. resolves #119

### CODE REVIEW

File: www/js/con-author.js [hunk 1]
```diff
@@ -24,11 +24,12 @@ define([
 	'dojo/dom',
 	'dojo/dom-class',
 	'dijit/registry',
+	'dojo/NodeList-dom',
 	'./controller',
 	"./equation",
 	"./typechecker",
 	"dojo/domReady!",
-], function(array, declare, lang, style, keys, ready, on, memory, aspect, dom, domClass, registry, controller, equation, typechecker){
+], function(array, declare, lang, style, keys, ready, on, memory, aspect, dom, domClass, registry, domList, controller, equation, typechecker){
```
Comment (riteshsamala) [inline www/js/con-author.js]: What is this module used for exactly ?

File: www/js/con-author.js [hunk 2]
```diff
@@ -178,10 +179,13 @@ define([
 				return this.disableHandlers || this.handleRoot(checked);
 			}));
 
-			var variableTypeToggle = dojo.query("input[type=radio][name=variableType]");
-			variableTypeToggle.on('change', lang.hitch(this, function(){
-				return this.disableHandlers || this.handleVariableType(event);
-			}));
+			debugger;
```
Comment (riteshsamala) [inline www/js/con-author.js]: I think this debugger keyword can be removed now

File: www/js/con-author.js (hunk 1 above)
Comment (sachingrover211) [inline www/js/con-author.js]: domList is the return object of dojo/query. It is used for defining forEach that I use.

File: www/js/con-author.js (hunk 2 above)
Comment (sachingrover211) [inline www/js/con-author.js]: done.

**Comment by sachingrover211** [comment]
@riteshsamala should I merge the pull request now?

**Pull request merged**